### PR TITLE
Anaconda Workflow

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#100 

### Added
- Anaconda workflow
### Changed
- Anaconda installation instruction added to `README.md`
- Package's dependencies added to `meta.yaml`

#### Any other comments?

**Do not merge this PR right before/after #104 due to probable silent merge conflicts.**

### CODE REVIEW

**Comment by AHReccese** [comment]
@sadrasabouri LGTM, Thank you so much for your work.
According to this [issue](https://github.com/bridgecrewio/checkov/issues/4127) it seems that the default access is `write-all` and sometimes(as discussed in the thread) in some projects suddenly this warning is getting raised and it is ok, even if you have not changed anything, so, I think it does address questions you raised in the last meeting, I tried to downgrade the workflow's access level from default `write-all` to `read-all` and it is now ok.

**Review by AHReccese — APPROVED** [review]

**Review by AHReccese — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@sadrasabouri, could you please rebase this PR and resolve any conflicts? Then ping me for the final review. 🔥

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Anaconda workflow
+- `README.md` modified
```
Comment (sepandhaghighi) [inline CHANGELOG.md:10]: Move this line to `Changed` section

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- `meta.yaml` modified
```
Comment (sepandhaghighi) [inline CHANGELOG.md:11]: Move this line to `Changed` section

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri I think something went wrong during the rebase process. I can't see any changes in `meta.yml`.

**Comment by sadrasabouri** [comment]
> @sadrasabouri I think something went wrong during the rebase process. I can't see any changes in `meta.yml`.

`meta.yml` has been changed in [last PR](https://github.com/openscilab/pymilo/pull/104/files#diff-2bcb57e74c1bcda3739d1c3a6d835ef6b67b384403dfe6b1a8ef1eee742f590cR21-R23).

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**